woocommerce/woocommerce-android
# Track event when CTA is tapped in the banner

Closes: #6925 


### Description


This PR adds tracking when the upsell card reader banner CTA is clicked on the payments, order list, and settings screen.

event name: `woocommerceandroid_feature_card_cta_tapped`

properties:

**For the payment screen:**

source : payment_methods
campaign_name : upsell_card_readers

**For the order list screen:**

source : order_list
campaign_name : upsell_card_readers

**For the settings screen:**

source : settings
campaign_name : upsell_card_readers

### Testing instructions


Tracking in the Payments screen

1. Navigate to the order eligible for IPP
2. Notice the Banner
3. Tap on Purchase card reader CTA and ensure the event is being tracked in the logs


Tracking in the Order list screen

1. Navigate to the Order list screen
2. Notice the Banner
3. Tap on Purchase card reader CTA and ensure the event is being tracked in the logs

Tracking in the Settings screen
1. Navigate to the Settings screen
2. Notice the Banner
3. Tap on Purchase card reader CTA and ensure the event is being tracked in the logs
